hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -437,6 +429,23 @@ public void createBucket(
     ozoneManagerClient.createBucket(builder.build());
   }
 
+  /**
+   * Helper function to get deafult acl list for current user.
+   *
+   * @return listOfAcls
+   * */
+  private List<OzoneAcl> getAclList() {
```

Comment:
Can we name this like: getDefaultAcls() and put it into a helper function of a comnon util class outside RpcClient?

Reply:
this helper function uses ugi, userRights and groupRights variables which are internal to this class.